src/vm.rs

```diff
@@ -763,7 +781,7 @@ impl VM {
 
                         Value::ExternFunction(slot) => {
                             let extern_func = self.extern_functions[*slot];
-                            extern_func(&self.stack[new_base+1..], false)
+                            extern_func(&self.stack[new_base+1..], true)
```

Comment:
No idea why this was wrong honestly! We've never type checked external functions?